_cities/milan.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Milan
+location: "Milan, Italy"
+tagline: "Pizza, code and fine wine!"
+mailing_list: team.milan
+does:
+  workshops
+members:
+  - alberto: Alberto Massa
+  - giorgia
+  - giorgio: Giorgio Mazzucchelli
+  - pietro909
```

Comment:
I'm registered as a forum user and in the mailing list since 2012. I don't know whether I should do something more.
For me is good to have the direct mail ```pietro909@opentechschool``` in order to keep the communication fluent.